LorafyAPI/Controllers/EndDeviceController.cs

```diff
@@ -1,20 +1,24 @@
-﻿using LorafyAPI.Models;
+﻿using LazyCache;
+using LorafyAPI.Models;
```

Comment:
Why have you imported Models?

Reply:
Because MQTTJsonMessage didn't work otherwise.